modules/accredited_representative_portal/app/services/query_params_schema.rb

```diff
@@ -0,0 +1,9 @@
+require 'dry-schema'
+
+QueryParamsSchema = Dry::Schema.Params do
```

Comment:
Let's rename this so it's more specific to POA Requests and the let's supply all the default values in here instead of the filter service.